javascript_nodejs/19.bot-authentication/bot.js

```diff
@@ -0,0 +1,148 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+
+const { ActivityTypes, MessageFactory } = require('botbuilder');
+const { ChoicePrompt, DialogSet, WaterfallDialog, OAuthPrompt } = require('botbuilder-dialogs');
+const CONNECTION_NAME = "";
```

Comment:
This should have a comment explaining what it is and what are acceptable values.